Author: Michael Biebl <biebl@debian.org>
Description: Support detection of libgmime 2.6 as well.
Bug: http://bugs.debian.org/663998

Included in upstream branch 1.8 . Should be in 1.8.12 .

Index: asterisk-1.8.8.2~dfsg/configure.ac
===================================================================
--- asterisk-1.8.8.2~dfsg.orig/configure.ac	2011-10-18 23:26:48.000000000 +0200
+++ asterisk-1.8.8.2~dfsg/configure.ac	2012-03-14 22:52:09.629478712 +0100
@@ -2019,7 +2019,7 @@
 AST_EXT_TOOL_CHECK([GMIME], [gmime-config], [], [], [#include <gmime/gmime.h>], [gboolean q = g_mime_check_version(0,0,0);])
 if test "x${PBX_GMIME}" = "x0" -a "${PKGCONFIG}" != "No"; then
 	# Later versions of GMime use pkg-config
-	for ver in 2.0 2.2 2.4; do
+	for ver in 2.0 2.2 2.4 2.6; do
 		if ! ${PKGCONFIG} --exists gmime-$ver; then
 			continue
 		fi
